public/mobile-app/src/routes/+page.svelte

```diff
@@ -1,276 +1,93 @@
+    '88d6fc32244b89e2617388fb111e668fec7b7383c841a08eefbd58fd11637eec'
+  const STATE = 'stateazertyuiopqsdfghjklmwxcvbn012345'
+  const NONCE = 'nonceazertyuiopqsdfghjklmwxcvbn012345'
+  const FC_URL = 'https://fcp-low.sbx.dev-franceconnect.fr'
+  const AUTHORIZATION_FC_PATH = '/api/v2/authorize'
+
+  const query = {
+    scope:
+      'openid given_name family_name preferred_username birthdate gender birthplace birthcountry sub email given_name_array',
+    redirect_uri: `${FS_URL}${DATA_CALLBACK_FS_PATH}`,
+    response_type: 'code',
+    client_id: DATA_CLIENT_ID,
+    state: STATE,
+    nonce: NONCE,
+    acr_values: 'eidas1',
+    prompt: 'login',
   }
-}
 
-const askForNotificationPermission = async () => {
-  const permissionGranted = await Notification.requestPermission()
-  const registration = await navigator.serviceWorker.ready
-  if (!permissionGranted || !registration) {
-    console.log(
-      'No notification: missing permission or missing service worker registration'
-    )
-    return
-  }
-  await updateButtonsStates()
-}
+  const url = `${FC_URL}${AUTHORIZATION_FC_PATH}`
+  const params = new URLSearchParams(query).toString()
 
-const registerWithAmi = async () => {
-  if (registerEmailInputValue !== '') {
-    userEmailValidationStatus = ''
+  window.location.href = `${url}?${params}`
 
-    const pushSubURL = pushSubscription.endpoint
-    const pushSubAuth = pushSubscription.toJSON().keys.auth
-    const pushSubP256DH = pushSubscription.toJSON().keys.p256dh
-
-    const payload = {
-      subscription: {
-        endpoint: pushSubURL,
-        keys: {
-          auth: pushSubAuth,
-          p256dh: pushSubP256DH,
-        },
-      },
-      email: registerEmailInputValue,
-    }
-    console.log('payload:', payload)
-    isRegisteredWithAmi = true
-    registrationStatus = "En cours d'enregistrement..."
-
-    const response = await fetch(`${PUBLIC_API_URL}/api/v1/registrations`, {
-      method: 'POST',
-      body: JSON.stringify(payload),
-    })
-    console.log('response:', response)
-    isRegisteredWithAmi = false
-    if (response.status < 400) {
-      registrationStatus = 'Enregistrement réussi !'
-      const registration = await response.json()
-      userIdState = registration.user_id
-      userEmailState = registerEmailInputValue
-      userNotifications = []
-      window.localStorage.setItem('userIdState', userIdState.toString())
-      window.localStorage.setItem('emailLocalStorage', userEmailState)
-    } else {
-      registrationStatus = `error ${response.status}: ${response.statusText}, ${response.body}`
-    }
-  } else {
-    userEmailValidationStatus = 'Veuillez remplir le champ Email'
-  }
+  return Response.redirect(`${url}?${params}`)
 }
 </script>
 
 <div>
-{#if !isAppInstalled}
-  <Installation />
-{:else}
-	<div>
-		{#if userEmailState}
-			<h1>Bienvenue {userEmailState} sur l'application AMI</h1>
-		{:else}
-			<h1>Bienvenue sur l'application AMI</h1>
-		{/if}
+{#if !isFranceConnected}
```

Comment:
As proposed above, maybe simply
```suggestion
{#if !userinfo}
```

Reply:
Answered with https://github.com/numerique-gouv/ami-notifications-api/pull/75#discussion_r2293070265